package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@faker-js/faker": "^8.3.1",
-    "@mapeo/schema": "^3.0.0-next.16",
+    "@mapeo/schema": "file:mapeo-schema-3.0.0-next.16.tgz",
```

Comment:
See <https://github.com/digidem/mapeo-schema/pull/191>. I'll deploy a new version of the dependency before merging this PR.